src/harness/fake_sas.py

```diff
@@ -409,16 +409,40 @@ def do_GET(self):
     self.wfile.write(json.dumps(response))
 
 
-def RunFakeServer(version, is_ecc):
+def RunFakeServer(version, is_ecc, ca_cert, verify_crl):
   FakeSasHandler.SetVersion(version)
   if is_ecc:
     assert ssl.HAS_ECDH
+  if ca_cert is not None:
+    try:
+      with open(os.path.join('certs', ca_cert)) as file_handle:
+        ca_cert_data = file_handle.read()
+    except IOError:
+      print "%s is not exist in certs path"
+    return
   server = HTTPServer(('localhost', PORT), FakeSasHandler)
-  server.socket = ssl.wrap_socket(
+
+  if verify_crl:
```

Comment:
I don't think it's good to have two independent code paths for initializing SSL.  Can you eliminate the `ssl.wrap_socket` branch, and just create an `ssl_context` with varying options?

Reply:
Changed   based on your suggestion and maintain the single branch